_posts/Getting Started/2020-07-14-create-flink-project-on-streaming-data-platform.md

```diff
@@ -20,39 +20,33 @@ This is a general guide for creating and setting up Flink job on Dell EMC Stream
 Apache Flink is the embedded analytics engine in the Dell EMC Streaming Data Platform which provides a seamless way for development teams to deploy analytics applications. This post describes the general instruction of using Apache Flink<sup>®</sup> applications with the Streaming Data Platform.
 
 ## Instructions
-##### A. Remove `createScope` method from the code
-**1.** If you have used `createScope` method in Pravega standalone mode. Please **comment out** from your code since the SDP does not allow to create a scope from the code.
+##### 1. Remove `createScope` method from the code
+**A.** If you have used `createScope` method in Pravega standalone mode. Please **comment out** from your code since the SDP does not allow to create a scope from the code.
 
-##### B. Create Projects
-**1.** Log in to the Dell EMC Streaming Data Platform and click the **Analytics** icon to navigate to the Analytics Projects view which lists the projects you have access to. 
+##### 2. Create Projects
+**A.** Log in to the Dell EMC Streaming Data Platform and click the **Analytics** icon to navigate to the Analytics Projects view which lists the projects you have access to. 
 ![Analytics-Project]({{site.baseurl}}/assets/heliumjk/images/post/flink-sdp-setup/analytics-project.png)
 **Note:** If you cannot log in to the Dell EMC Streaming Data Platform, please use ```kubectl get ingress -A``` to find all ingress and add to your **/etc/hosts** file.  
 
-**2.** Click **Create Project** button to start with a new project by typing the project name, description and required volume size. This will also automatically create a scope or namespace for your project in Pravega. 
-![Create-Project]({{site.baseurl}}/assets/heliumjk/images/post/flink-sdp-setup/create-project.png)
+**B.** Click **Create Project** button to start with a new project by typing the project name, description and required volume size. This action will also automatically create a scope or namespace for your project in Pravega. 
+![Create-Project]({{site.baseurl}}/assets/heliumjk/images/post/flink-sdp-setup/create-project.png)      
 
-##### C. Create Flink Clusters
-**1.** To create a Flink cluster using the Dell EMC Streaming Data Platform UI, navigate to **Analytics -> *project-name* > Flink Clusters**.  
-![Flink-Cluster]({{site.baseurl}}/assets/heliumjk/images/post/flink-sdp-setup/flink-cluster.png)
-
-**2.** Click **Create Flink Cluster** and complete the cluster configuration fields.
-![Create-Flink-Cluster]({{site.baseurl}}/assets/heliumjk/images/post/flink-sdp-setup/create-flink-cluster.png)
 
-##### D. Upload artifact to SDP
-###### Option I: Using the Dell EMC Streaming Data Platform UI
-**1.** Navigate to **Analytics > Analytics Projects > *project-name* > Artifacts**
+##### 3. Upload artifact to SDP
```

Comment:
Rename this to "Upload Flink artifact to SDP" and insert some text that says something like "next, you must upload your Flink job artifact[s] to the SDP maven repository"